docs/tutorials/gluon/learning_rate_schedules_advanced.md

```diff
@@ -0,0 +1,308 @@
+
+
+```python
+schedule = mx.lr_scheduler.MultiFactorScheduler(step=[250, 750, 900], factor=0.5)
+schedule.base_lr = 1
+schedule = LinearWarmUp(schedule, start_lr=0, length=250)
+plot_schedule(schedule)
+```
+
+
+![png](https://raw.githubusercontent.com/dmlc/web-data/master/mxnet/doc/tutorials/lr_schedules/adv_warmup.png) <!--notebook-skip-line-->
+
+
+### Cool-Down
+
+Similarly, we could add a linear cool-down period to our schedule and this is used in the "1-Cycle" schedule proposed by [Leslie N. Smith, Nicholay Topin (2017)](https://arxiv.org/abs/1708.07120) to train neural networks very quickly in certain circumstances (coined "super-convergence"). We reduce the learning rate from its value at `start_idx` of `schedule` to `finish_lr` over a period of `length`, and then maintain `finish_lr` thereafter.
+
+
+```python
+class LinearCoolDown():
+    def __init__(self, schedule, finish_lr, start_idx, length):
+        """
+        schedule: a pre-initialized schedule (e.g. TriangularSchedule(min_lr=0.5, max_lr=2, cycle_length=500))
+        finish_lr: learning rate used at end of the cool-down (float)
+        start_idx: iteration to start the cool-down (int)
+        length: number of iterations used for the cool-down (int)
+        """
+        self.schedule = schedule
+        # calling mx.lr_scheduler.LRScheduler effects state, so calling a copy
+        self.start_lr = copy.copy(self.schedule)(start_idx)
+        self.finish_lr = finish_lr
+        self.start_idx = start_idx
+        self.finish_idx = start_idx + length
+        self.length = length
+    
+    def __call__(self, iteration):
+        if iteration <= self.start_idx:
+            return self.schedule(iteration)
+        elif iteration <= self.finish_idx:
+            return (iteration - self.start_idx) * (self.finish_lr-self.start_lr)/(self.length) + self.start_lr
+        else:
+            return self.finish_lr
+```
+
+As an example, we apply learning rate cool-down to a `MultiFactorScheduler`. Starting the cool-down at iteration 1000, we reduce the learning rate linearly from 0.125 to 0.001 over 500 iterations, and hold the learning rate at 0.001 after this.
+
+
+```python
+schedule = mx.lr_scheduler.MultiFactorScheduler(step=[250, 750, 900], factor=0.5)
+schedule.base_lr = 1
+schedule = LinearCoolDown(schedule, finish_lr=0.001, start_idx=1000, length=500)
+plot_schedule(schedule)
+```
+
+
+![png](https://raw.githubusercontent.com/dmlc/web-data/master/mxnet/doc/tutorials/lr_schedules/adv_cooldown.png) <!--notebook-skip-line-->
+
+
+#### 1-Cycle: for "Super-Convergence"
+
+Cool-down is used in the "1-Cycle" schedule proposed by [Leslie N. Smith, Nicholay Topin (2017)](https://arxiv.org/abs/1708.07120) to train neural networks very quickly in certain circumstances (coined "super-convergence"). We implement a single and symmetric cycle of the triangular schedule above (i.e. `inc_fraction=0.5`), followed by a cool-down period of `cooldown_length` iterations.
```

Comment:
Super-convergence is also introduced in the previous section's description.  Seems a bit redundant here.